### PR TITLE
Fix NPEs

### PR DESCRIPTION
**Pull request by anatoliy-balakirev**
Found a couple of NPEs while testing something else.

### CODE REVIEW

File: specs/spring-cloud-contract-spec-java/src/main/java/org/springframework/cloud/contract/spec/internal/OptionalProperty.java [hunk 1]
```diff
@@ -44,8 +44,15 @@ public String optionalPattern() {
 	}
 
 	public String value() {
-		return this.value instanceof RegexProperty
-				? ((RegexProperty) this.value).pattern() : this.value.toString();
+		if (this.value == null) {
+			return "";
```
Comment (anatoliy-balakirev) [inline specs/spring-cloud-contract-spec-java/src/main/java/org/springframework/cloud/contract/spec/internal/OptionalProperty.java]: Not sure if this is what needs to be returned here...

Comment (marcingrzejszczak) [reply specs/spring-cloud-contract-spec-java/src/main/java/org/springframework/cloud/contract/spec/internal/OptionalProperty.java]: There should be no such option that `null` is passed in. We can leave `""`

**Comment by marcingrzejszczak** [comment]
Can you rebase against master and try to build it again with the new base image for circleci?

**Comment by anatoliy-balakirev** [comment]
@marcingrzejszczak seems to be the same after rebase.

**Comment by marcingrzejszczak** [comment]
Jacoco was the culprit. Please rebase and it should work fine.

**Pull request merged**